gr-fec/grc/variable_cc_decoder_def_list.block.yml

```diff
@@ -48,12 +48,12 @@ parameters:
     default: '-1'
 -   id: mode
     label: Streaming Behavior
-    dtype: enum
-    options: [fec.CC_STREAMING, fec.CC_TERMINATED, fec.CC_TAILBITING, fec.CC_TRUNCATED]
+    dtype: raw
+    options: ['fec.CC_STREAMING', 'fec.CC_TERMINATED', 'fec.CC_TAILBITING', 'fec.CC_TRUNCATED']
     option_labels: [Streaming, Terminated, Tailbiting, Truncated]
 -   id: padding
     label: Byte Padding
-    dtype: enum
+    dtype: bool
     default: 'False'
     options: ['False', 'True']
```

Comment:
`options` and `option_labels` are ignored for `bool` types, and can be removed.

Reply:
Manually amended.